setup.py

```diff
@@ -27,50 +27,18 @@ def c_includes(parent, depth=1):
     return includes
 
 
-PACKAGENAME = 'stsci.stimage'
 SOURCES = c_sources('src')
 INCLUDES = c_includes('include') + c_includes('src') + [numpy_includes()]
 
 
 setup(
-    name=PACKAGENAME,
     use_scm_version=True,
     setup_requires=['setuptools_scm'],
```

Comment:
Might as well move the rest of the metadata to setup.cfg?

Reply:
There's actually a good reason to keep `setuptools_scm` related metadata in `setup()`, as it makes it more backwards compatible with older `setuptools`.  I don't know how long we'll have to do this?  Maybe we don't have to anymore?

Reply:
Ah, and it looks like setuptools 42 was the point that this started working.  Maybe I should roll back that requirement given that there's lots of legacy code that still uses this package.  For maximal compatibility.